.teamcity/settings.kts

```diff
@@ -970,7 +970,7 @@ object RunCalypsoE2eDesktopTests : BuildType({
 				set -x
 
 				mkdir -p screenshots
-				find test/e2e/temp -path '*/screenshots/*' -print0 | xargs -r -0 mv -t screenshots
+				find test/e2e/temp -type f -path '*/screenshots/*' -print0 | xargs -r -0 mv -t screenshots
```

Comment:
Without this, it will try to move `screenshots/video/` and then `screenshots/video/file.mpg`, which will fail because it was moved as part of the `video` dir